utils/misc/misc.go

```diff
@@ -85,6 +85,17 @@ type RudderError struct {
 	Code              int
 }
 
+type Pair struct {
+	Key   string
+	Value float64
+}
+
+type PairList []Pair
+
+func (p PairList) Len() int           { return len(p) }
+func (p PairList) Swap(i, j int)      { p[i], p[j] = p[j], p[i] }
+func (p PairList) Less(i, j int) bool { return p[i].Value < p[j].Value }
```

Comment:
Not sure if `misc` is the correct place for this. Could be moved to router package or sub-package since it is router specific at the moment.

Reply:
Updated this